diff --git a/src/vs/workbench/workbench.desktop.main.ts b/src/vs/workbench/workbench.desktop.main.ts
index 9bc4dbca..2c969c7e 100644
--- a/src/vs/workbench/workbench.desktop.main.ts
+++ b/src/vs/workbench/workbench.desktop.main.ts
@@ -119,5 +119,2 @@ import './contrib/localization/electron-browser/localization.contribution.js';
 
-// Explorer
-import './contrib/files/electron-browser/fileActions.contribution.js';
-
 // CodeEditor Contributions
@@ -125,21 +122,2 @@ import './contrib/codeEditor/electron-browser/codeEditor.contribution.js';
 
-// Debug
-import './contrib/debug/electron-browser/extensionHostDebugService.js';
-
-// Extensions Management
-import './contrib/extensions/electron-browser/extensions.contribution.js';
-import './contrib/extensions/electron-browser/devtoolsExtensionHost.contribution.js';
-
-// Issues
-import './contrib/issue/electron-browser/issue.contribution.js';
-
-// Surveys
-import './contrib/surveys/browser/survey.contribution.js';
-
-// Process Explorer
-import './contrib/processExplorer/electron-browser/processExplorer.contribution.js';
-
-// Remote
-import './contrib/remote/electron-browser/remote.contribution.js';
-
 // Terminal
@@ -150,8 +128,2 @@ import './contrib/themes/browser/themes.test.contribution.js';
 import './services/themes/electron-browser/themes.contribution.js';
-// User Data Sync
-import './contrib/userDataSync/electron-browser/userDataSync.contribution.js';
-
-// Tags
-import './contrib/tags/electron-browser/workspaceTagsService.js';
-import './contrib/tags/electron-browser/tags.contribution.js';
 // Performance
@@ -165,8 +137,2 @@ import './contrib/externalTerminal/electron-browser/externalTerminal.contributio
 
-// Webview
-import './contrib/webview/electron-browser/webview.contribution.js';
-
-// Browser
-import './contrib/browserView/electron-browser/browserView.contribution.js';
-
 // Splash
@@ -174,14 +140,2 @@ import './contrib/splash/electron-browser/splash.contribution.js';
 
-// Local History
-import './contrib/localHistory/electron-browser/localHistory.contribution.js';
-
-// Merge Editor
-import './contrib/mergeEditor/electron-browser/mergeEditor.contribution.js';
-
-// Multi Diff Editor
-import './contrib/multiDiffEditor/browser/multiDiffEditor.contribution.js';
-
-// Remote Tunnel
-import './contrib/remoteTunnel/electron-browser/remoteTunnel.contribution.js';
-
 // Chat
@@ -202,8 +156,2 @@ import './contrib/mcp/electron-browser/mcp.contribution.js';
 
-// Policy Export
-import './contrib/policyExport/electron-browser/policyExport.contribution.js';
-
-// Keybindings Export
-import './contrib/keybindingsExport/electron-browser/keybindingsExport.contribution.js';
-
 // System-wide (OS global) Keybindings
